portfolio/src/main/webapp/scripts/login.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Shows feedback on login status
+ */
+async function showCommentFeedback() {
+  try {
+    const response = await fetch('/create-new-comment');
+    const loginStatus = await response.text();
+    document.getElementById('login-feedback').innerText = loginStatus;
```

Comment:
Just check whether loginStatus is true/false and decide here what the rendering should look like based on that.

Reply:
Got it!